.github/workflows/build-test-virtual.yml

```diff
@@ -0,0 +1,34 @@
+name: "Build/test (virtual)"
+on: [push]
+jobs:
+  build-test:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: mcr.microsoft.com/ccf/app/dev:3.0.9-virtual
+      env:
+        # Helps to distinguish between CI and local builds.
+        SCITT_CI: 1
+        # Additional CMake flags.
+        PLATFORM: virtual
+        CMAKE_BUILD_TYPE: Debug
+        ENABLE_PREFIX_TREE: ON
+        ENABLE_CLANG_TIDY: ON
```

Comment:
Do you think it could make sense to enable performance tests as well through the corresponding environment variable?

Reply:
They are/were enabled in Azure pipelines but only on the proper SGX hardware https://github.com/microsoft/scitt-ccf-ledger/blob/main/.pipelines/azure-pipelines.yml I was thinking about enabling it here but is feels like overkill and a bit misleading to do it in the PRs using virtual